lib/dotiw/locale/ar.yml

```diff
@@ -38,16 +38,67 @@ ar:
         other: "%{count} أسيوع"
       months:
         zero: شهر
-        one: شهر واحد 
+        one: شهر واحد
         two: شهرين
         few: "%{count} أشهر"
         many: "%{count} شهراً"
         other: "%{count} شهر"
       years:
         zero: عام
-        one: عام واحد 
+        one: عام واحد
         two: عامين
         few: "%{count} أعوام"
         many: "%{count} عاماً"
         other: "%{count} عام"
       less_than_x: "%{distance} أقل من"
+  dotiw_compact:
```

Comment:
I am told by a friend-of-a-friend that Arabic does not have "abbreviations" per se.

```
First, let me say my Arabic isn’t good enough to provide an actual answer to your question. A native speaker might totally contradict me and they’d be right. However, Arabic very rarely uses abbreviations like that.

In one sense, the words are already abbreviated, because they lack the short vowels.

The only time I can recall seeing “abbreviations” is in the formation of Arabic acronyms (FATAH, HAMAS, etc) which is its own art and not really the same thing.

So: no real answer, but maybe some reassurance that there may not be not a commonly recognized equivalent.

Example: here’s an Arabic calendar. Where we would normally put SMTWTF, they had to squeeze in the entire name of each day.
```